cmp/grid/GridModel.js

```diff
@@ -1154,6 +1165,12 @@ export class GridModel extends HoistModel {
         this.isEditing = false;
     }
 
+    @debounced(500)
+    @action
+    updateIsInEditingMode(isEditing) {
```

Comment:
Either move to implemenation area, or do inline in reaction, using the debounce argument to addReaction
(I would probably prefer the latter)